services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_task.py

```diff
@@ -54,10 +64,15 @@ def setup(app: FastAPI):
     if any(
         s is None
         for s in [
-            app_settings.AUTOSCALING_NODES_MONITORING,
             app_settings.AUTOSCALING_EC2_ACCESS,
             app_settings.AUTOSCALING_EC2_INSTANCES,
         ]
+    ) or all(
+        s is None
+        for s in [
```

Comment:
This kind of answers my question before about the x-or  ...
Nonetheless, i think we shold be more strict upon setup regarding "optionality". It is partially my fault with the "plugins design policy". I think we should support only a certain range of states specially at startup and fail otherwise. The more states we allow the more combinations so we should not allow unnecessary states. I wonder whether a "nothing will happen" state is necessary.